python/tests/test_highlevel.py

```diff
@@ -1478,6 +1478,94 @@ def test_nbytes(self, tmp_path, ts_fixture):
         )
         assert nbytes == ts_fixture.nbytes
 
+    def test_equals(self, ts_fixture):
```

Comment:
Can't we just not have `ts_fixture`` as a parameter then, ie.`` def test_equals(self)`` ?

Reply:
Opps - left in from when I tried it out.